.github/workflows/build_dev_container.sh

```diff
@@ -0,0 +1,22 @@
+#!/usr/bin/env bash
+
+set -x -e
+
+df -h
+docker info
+# to get more disk space
+rm -rf /usr/share/dotnet &
```

Comment:
Could this be a problem if users test the dev_container image builds process locally (i.e. PRs over dev_container recepits itself)?

Reply:
Yeah which is why I moved it into `.github/workflows` it is not meant to be built locally. We could make it so it calls a script in `tools/` but then the user would need download the base image which is almost as large so I don't see a lot of benefit